xl_auth/token/models.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""Token models."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from codecs import getencoder
+from datetime import datetime, timedelta
+from os import urandom
+
+from ..database import Column, Model, SurrogatePK, db, relationship
+
+
+class Token(SurrogatePK, Model):
+    """An OAuth2 Bearer token."""
+
+    __tablename__ = 'tokens'
+    user_id = Column(db.Integer, db.ForeignKey('users.id', ondelete='CASCADE'), nullable=False)
+    user = relationship('User')
+
+    client_id = Column(db.String(64), db.ForeignKey('clients.client_id', ondelete='CASCADE'),
```

Comment:
Use improved `reference_col` instead of `Column`?

Reply:
Fixed.